pkg/apis/admissionregistration/v1alpha1/defaults.go

```diff
@@ -49,3 +49,11 @@ func SetDefaults_MatchResources(obj *admissionregistrationv1alpha1.MatchResource
 		obj.ObjectSelector = &selector
 	}
 }
+
+// SetDefaults_ParamRef sets defaults for ParamRef
```

Comment:
Do we make it required or default to deny? I thought we make it required?

Reply:
For v1alpha1 we default to Deny, otherwise required.